src/auth/session/storage/custom.ts

```diff
@@ -102,6 +103,8 @@ export class CustomSessionStorage implements SessionStorage {
   }
 
   public async findSessionsByShop(shop: string): Promise<SessionInterface[]> {
+    sanitizeShop(shop, true);
```

Comment:
Good point, we should probably still use the sanitized value in case we ever want to actually _alter_ it (e.g. add a domain if it's missing, or some such).